scripts/nn/test/compare_backends/run_tests.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
Can you update this to `#!/usr/bin/env bash` to use the user's preferred bash, rather than a hardcoded system version?  Also applies to the other shell scripts.